data_linter/command_line.py

```diff
@@ -0,0 +1,13 @@
+from data_linter import run_validation
+import argparse
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-c", "--config_path", help="Path to a config.yaml")
+    args = parser.parse_args()
+    config_path = args.config
```

Comment:
Think this should be `args.config_path`

Reply:
Otherwise i get `AttributeError: 'Namespace' object has no attribute 'config'`

Reply:
🤦🏻‍♂️